internal/datastore/crdb/keys.go

```diff
@@ -1,10 +1,17 @@
 package crdb
 
-import "strings"
+import (
+	"context"
+	"strings"
+
+	"google.golang.org/grpc/metadata"
+)
+
+const overlapMDKey = "spicedb-overlap"
```

Comment:
we should define this in authzed-go in here: https://github.com/authzed/authzed-go/blob/main/pkg/requestmeta/requestmeta.go

Reply:
updated